<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCategoryList</class>
 <widget class="QDialog" name="DlgCategoryList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbCategoryList">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>761</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="twCategoryList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>901</width>
     <height>571</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>#</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Code</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="tbInvertSelection">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource/icons.qrc">
     <normaloff>:/button/images/svg/retweet-solid.svg</normaloff>:/button/images/svg/retweet-solid.svg</iconset>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbSelection">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>10</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource/icons.qrc">
     <normaloff>:/button/images/svg/check-solid.svg</normaloff>:/button/images/svg/check-solid.svg</iconset>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbUnselection">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>10</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource/icons.qrc">
     <normaloff>:/button/images/svg/times-solid.svg</normaloff>:/button/images/svg/times-solid.svg</iconset>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbSave">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>10</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../resource/icons.qrc">
     <normaloff>:/button/images/svg/save-solid.svg</normaloff>:/button/images/svg/save-solid.svg</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="tbRefresh">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>10</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource/icons.qrc">
     <normaloff>:/button/images/svg/sync-alt-solid.svg</normaloff>:/button/images/svg/sync-alt-solid.svg</iconset>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource/icons.qrc"/>
 </resources>
 <connections/>
</ui>
